src/interfaces/appstore.js

```diff
@@ -298,6 +298,21 @@ module.exports = function(app) {
   }
 
   function installSKModule(module, version) {
+    if (module === 'signalk-server') {
```

Comment:
I think it is time to add `isTheServerModule()` in modules.ts.

https://github.com/SignalK/signalk-server/blob/master/src/modules.ts#L177
https://github.com/SignalK/signalk-server/blob/master/src/interfaces/appstore.js#L58